src/react-image-lightbox.js

```diff
@@ -1180,6 +1189,13 @@ class ReactImageLightbox extends Component {
     this.getSrcTypes().forEach(srcType => {
       const type = srcType.name;
 
+      // there is no error when we try to load it initially
+      if (props[type]) {
```

Comment:
Maybe add another condition to check if the `this.state.loadErrorStatus[type]` is `true` before setting it to `false` here. I want to avoid unnecessary rerenders due to `this.setState`